CMakeLists.txt

```diff
@@ -277,6 +277,7 @@ target_include_directories(${PDAL_BASE_LIB_NAME}
 target_link_libraries(${PDAL_BASE_LIB_NAME}
     PRIVATE
         ${CMAKE_THREAD_LIBS_INIT}
+        ${CMAKE_DL_LIBS}
```

Comment:
Why did this move from the util lib to the base lib?

Reply:
Because it wasn't used there, AFAICT.